src/SwitchPermissions.js

```diff
@@ -56,6 +58,6 @@ export default class SwitchPermissions extends Component {
             return createElement(loading);
         }
 
-        return cloneElement(match, props);
+        return <div>{React.Children.map(match, child => <FormField input={cloneElement(child)} {...props} />)}</div>;
```

Comment:
Not sure the `map` is a good idea here as it implies wrapping the children inside a parent element (`div`)